docs/about.md

```diff
@@ -0,0 +1,5 @@
+# About VS Code Extension
+
+Adds really basic syntax highlighting for djinni interface files.
+
+Get the extension in the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=cross-language-cpp.djinni-syntax-highlighting)
```

Comment:
we don't have that jet, or, we need to publish that ?

Reply:
I already created the publisher (`cross-language-cpp`). This link will be working once we published the plugin. Everything is perpared to publish with the pipeline to test if its working :)